tests/bson/bson-fromPHP-001.phpt

```diff
@@ -21,18 +21,29 @@ class MyDocument implements MongoDB\BSON\Serializable
     }
 }
 
+class MyPersistableDocument extends MyDocument implements MongoDB\BSON\Persistable
+{
+    public function bsonUnserialize(array $data): void
+    {
+        $this->data = $data;
+    }
+}
+
 $tests = array(
     array(1, 2, 3),
     array('foo' => 'bar'),
     (object) array(1, 2, 3),
     (object) array('foo' => 'bar'),
+    MongoDB\BSON\PackedArray::fromPHP([1, 2, 3]),
+    MongoDB\BSON\Document::fromPHP(['foo' => 'bar']),
```

Comment:
Does this test answer the earlier question I asked about `Persistable::bsonSerialize()` returning a PackedArray? Based on the expected output it looks like PackedArrays are silently encoded as documents and an ODM field appended (not desirable, IMO).

Reply:
Noted that you added exceptions below:

> Expected MyPersistableDocument::bsonSerialize() to return an array, stdClass, or MongoDB\BSON\Document, MongoDB\BSON\PackedArray given

I'll resolve this.